modules/enabling-seamless-image-access-and-efficient-use-of-multiple-registry-sources.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-seamless-image-access-and-efficient-use-of-multiple-registry-sources_{context}"]
+= Enabling seamless image access and efficient use of multiple registry sources
```

Comment:
Is "seamless" a crucial descriptor here? Seems sort of marketingish to me, but, again, my domain knowledge is weak.

Reply:
Perhaps just `Registry mirroring` is enough ("registry repository mirroring" is how it's referenced in [one of the headings](https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/updating_clusters/performing-a-cluster-update#images-configuration-registry-mirror-configuring_updating-restricted-network-cluster) of the OCP 4.16 docs)